### PR TITLE
Added `Author_image.png` into root base files when creating a pack.

### PR DESCRIPTION
**Pull request by obenacot**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [#36569](https://github.com/demisto/etc/issues/36569)

## Description
Added `Author_image.png` into root base files when creating a pack.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/pack_unique_files.py [hunk 1]
```diff
@@ -272,6 +273,36 @@ def validate_version_bump(self):
...
+        if not author_image.size == AUTHOR_IMAGE_DIMENSIONS:
+            self._add_error(Errors.author_image_fie_invalid_dimensions(), self.pack_meta_file)
+            return False
+        return True
+
+    def _is_author_image_size_valid(self, author_image_path) -> bool:
+        if not os.path.getsize(author_image_path) <= AUTHOR_IMAGE_SIZE:
+            self._add_error(Errors.author_image_fie_invalid_size(), self.pack_meta_file)
+            return False
+        return True
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/common/hook_validations/pack_unique_files.py:304]: We shouldn't validate these as part of the init. We only validate these kind of thing in the validate command

File: demisto_sdk/commands/common/tests/pack_unique_files_test.py [hunk 2]
```diff
@@ -484,8 +492,7 @@ def show(self, var):
         assert "Unable to find previous pack_metadata.json file - skipping price change validation" in \
                capsys.readouterr().out
 
-    @pytest.mark.parametrize('text, result', README_INPUT_RESULTS_LIST)
-    def test_validate_pack_readme_file_is_not_empty_partner(self, mocker, text, result):
+    def test_validate_pack_readme_file_is_not_empty_partner(self, mocker, result):
```
Comment (yaakovi) [inline demisto_sdk/commands/common/tests/pack_unique_files_test.py:495]: why did you change this?

**Review by yaakovi — CHANGES_REQUESTED** [review]
your build is failing

**Review by yaakovi — CHANGES_REQUESTED** [review]
you haven't fixed my original comments.
fix the LGTM alert

**Review by yaakovi — APPROVED** [review]
fix the conflicts and merge

**Pull request merged**